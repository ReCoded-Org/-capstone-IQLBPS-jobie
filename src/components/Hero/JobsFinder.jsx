import React from 'react'

function JobsFinder() {
  return (
    <div className="   bg-gray-200  flex-col justify-around  items-center py-16 ">
      <div>
        <div>
          <h1 className="text-3xl font-medium text-dark-gray leading-loose ">
            {' '}
            Jobs Finder
          </h1>
          <p className="decoration-solid pt-0 text-dark-gray leading-loose">
            {' '}
            Find the best jobe for you here
          </p>
        </div>
        <div className=" flex-col mt-2">
          <input
            type="search"
            placeholder="Type to search "
            className="rounded-3xl px-6 py-2 outline-none relative left-8"
          />
          <button
            type="button"
            className="py-2 bg-accent px-5 rounded-3xl text-white relative   hover:bg-red-500 "
          >
            {' '}
            Search
          </button>
        </div>
      </div>
    </div>
  );
}

export default JobsFinder