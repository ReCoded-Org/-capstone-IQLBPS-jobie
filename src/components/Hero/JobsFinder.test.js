import renderer from "react-test-renderer";
import JobsFinder from "./JobsFinder";

it("renders correctly when the component matches the snapshot", () => {
  const tree = renderer.create(<JobsFinder />).toJSON();
  expect(tree).toMatchSnapshot();
});
