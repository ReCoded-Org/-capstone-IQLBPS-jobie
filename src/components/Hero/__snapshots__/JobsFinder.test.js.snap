// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when the component matches the snapshot 1`] = `
<div
  className="App  bg-gray-200  flex-col justify-around  py-[50px]  "
>
  <div
    className="flex  flex-col"
  >
    <div>
      <h1
        className="text-3xl font-medium text-dark-gray leading-loose "
      >
         
        Jobs Finder
      </h1>
      <p
        className="decoration-solid pt-0 text-dark-gray leading-loose"
      >
         
        Find the best job for you here
      </p>
    </div>
    <div
      className=" flex-col mt-2"
    >
      <form
        onSubmit={[Function]}
      >
        <input
          className="rounded-3xl px-6 py-2 outline-none relative md:left-8"
          onChange={[Function]}
          placeholder="Type to search "
          type="search"
          value=""
        />
        <button
          className="py-2 bg-accent px-5 rounded-3xl text-white relative   hover:bg-red-500 "
          type="submit"
        >
           
          Search
        </button>
      </form>
    </div>
  </div>
</div>
`;
